plugins/destination/gremlin/client/client.go

```diff
@@ -0,0 +1,76 @@
+package client
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+
+	gremlingo "github.com/apache/tinkerpop/gremlin-go/v3/driver"
+	"github.com/cloudquery/plugin-sdk/plugins/destination"
+	"github.com/cloudquery/plugin-sdk/specs"
+	"github.com/rs/zerolog"
+)
+
+type Client struct {
+	destination.UnimplementedUnmanagedWriter
+	destination.DefaultReverseTransformer
+	logger     zerolog.Logger
+	spec       specs.Destination
+	pluginSpec Spec
+	client     *gremlingo.DriverRemoteConnection
+}
+
+var AnonT = gremlingo.T__
```

Comment:
why not using it directly?

Reply:
StackOverflow told me to. In previous incarnations this was used more than twice, now it's used just two times but it's easier to use/read still.